packages/core/src/components/ui/VirtualTable/VirtualTable.js

```diff
@@ -84,21 +112,91 @@ const VirtualTable = (props) => {
 }
 
 VirtualTable.propTypes = {
+  /**
+   * Data, which is rendering in VirtualTable
+   */
   data: PropTypes.oneOfType([PropTypes.array, PropTypes.object]),
-  columns: PropTypes.array, // eslint-disable-line
+  /**
+   * Array with all columns config
+   */
+  columns: PropTypes.arrayOf(
+    PropTypes.shape({
+      /**
+       * Name, which is showing
+       */
+      label: PropTypes.string,
+      /**
+       * Key from object of data
+       */
+      dataKey: PropTypes.string,
+      /**
+       * Width of colunm, in px
+       */
+      width: PropTypes.number,
+      /**
+       * Custom method of render column's data
+       */
+      cellRenderer: PropTypes.func,
+    }),
```

Comment:
this shape is missing other used props, some examples, so would it throw error if those other props are passed ?
`     flexGrow: 1,
      headerStyle: styles.PAIR,
      style: styles.PAIR,
      headerClassName: 'pair',`